portfolio/src/main/webapp/js/script.js

```diff
@@ -50,19 +50,40 @@ function makeChangeDescriptionCallBack(projectIndex) {
 }
 
 /**
- * Initialize map.
+ * Puts shawarma markers and adds event listeners to show description.
  */
+function putShawarmaMarkers() {
+    fetch('/shawarma', {method: 'GET'}).then(response => response.json()).then((shawarmaPlaces) => {
+        shawarmaPlaces.forEach(shawarmaPlace => {
+            const shawarmaMarker = new google.maps.Marker(
+            {position: {lat: shawarmaPlace.latitude, lng: shawarmaPlace.longitude}, map: map, title: shawarmaPlace.name});
```

Comment:
nit: this line is too long... the good thing is you can break it based on the structure. something like:

``` javascript
new google.maps.Marker({
  position: {
    lat: shawarmaPlace.latitude, 
    lng: shawarmaPlace.longitude
  }, 
  map: map,
  title: shawarmaPlace.name
});
```